B/bitshuffle/build_tarballs.jl

```diff
@@ -0,0 +1,37 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "bitshuffle"
+version = v"0.4.2"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/kiyo-masui/bitshuffle.git", "a60471d37a8cbbd8265dc8cfa83a9320abdcb590")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+
+cd bitshuffle/src
+$CC -O3 -std=c99 -fPIC --shared -o bitshuffle.${dlext} iochain.c bitshuffle_core.c
+install -D -t ${libdir} bitshuffle.${dlext}
+exit
```

Comment:
Can simplify a bit:
```suggestion
cd $WORKSPACE/srcdir/bitshuffle/src
mkdir -p "${libdir}"
cc -O3 -std=c99 -fPIC --shared -o "${libdir}/bitshuffle.${dlext}" iochain.c bitshuffle_core.c
```
Also, libraries are typically called `libNAME.${dlext}`, which I haven't changed here, but I recommend doing so for consistency with the rest of the world.